versioned_docs/version-1.0.0/install-kubeslice/kubeslice-cli/topology-configuration.mdx

```diff
@@ -85,6 +87,34 @@ The following table describes the worker chart configuration parameters used to
 | version    | Numeric        | The version of the chart. Leave this value blank for the latest version.         | Optional  |
 | values     | String         | The additional value that can be passed as --set arguments for **helm install**. | Optional  |
 
+## UI Chart Configuration Parameters
+The following table describes the UI chart configuration parameters used to create topology YAML.
+
+| Parameter  | Parameter Type | Description                                                                      | Required  |
+| ---------- | -------------- | -------------------------------------------------------------------------------- | --------- |
+| chart_name | String         | The name of the UI or enterprise chart.                                          | Mandatory |
+| version    | Numeric        | The version of the chart. Leave it blank for the latest version.                 | Optional  |
+| values     | String         | The additional value that can be passed as --set arguments for **helm install**. | Optional  |
+
+## Prometheus Chart Configuration parameters
+The following table describes the Prometheus chart configuration parameters used to create topology YAML.
+
+| Parameter  | Parameter Type | Description                                                                      | Required  |
+| ---------- | -------------- | -------------------------------------------------------------------------------- | --------- |
+| chart_name | String         | The name of the Prometheus chart.                                                | Mandatory |
+| version    | Numeric        | The version of the chart. Leave it blank for the latest version.                 | Optional  |
+| values     | String         | The additional value that can be passed as --set arguments for **helm install**. | Optional  |
+
+## Image Pull Secret Configuration Parameters
```

Comment:
These are not required in the OSS edition...

Reply:
- same as above

Reply:
We need to let the user know that it is only required with `enterprise-demo` in our docs (even if it's not there in the tool help description), How will the user know how to get the image pull secrets? Does the tool show up the registration form? 

It's also good to have this https://docs.avesha.io/documentation/enterprise/1.3.0/playground/playground-ent-kubeslice-demo-kind#enterprise-artifact-registration description added as a cross reference. 

cc @narmidm. 

I discussed this with Imran and he told me that it's good to let the user know about the Enterprise demo. 

We are retaining it but we should let the user know that UI charts, image pull charts and/or Prometheus charts are required only with the enterprise-demo option...